### DISCUSSION TITLE
Hi, I have some question about model package in main2.py

### DISCUSSION THREAD

**Discussion by ParkJeongHo-code** [yazdanimehdi/AttentionSiteDTI#2] [Q&A] [answered]
Hi, I'm master course student in Chungbuk national university in South Korea. I have some question about package that name is model existing in main2.py. I can't find the package. I was think the package should be install, since there are no code file that name is model.py in your repository. I wanna know what should i do for using model package.

**Comment by yazdanimehdi** [comment, marked as answer]
Hello,
Try importing DTITAG from model_abiliation.py. The model does not have any impact on the code, and you can go ahead and remove the import line.
I have corrected this in the main2.py.

**Reply by ParkJeongHo-code** [reply]
thank you for your answer. have a nice day
